Mathlib/Analysis/NormedSpace/Complemented.lean

```diff
@@ -88,57 +88,57 @@ def prodEquivOfClosedCompl (h : IsCompl p q) (hp : IsClosed (p : Set E))
   haveI := hp.completeSpace_coe; haveI := hq.completeSpace_coe
   refine' (p.prodEquivOfIsCompl q h).toContinuousLinearEquivOfContinuous _
   exact (p.subtypeL.coprod q.subtypeL).continuous
-#align subspace.prod_equiv_of_closed_compl Subspace.prodEquivOfClosedCompl
+#align subspace.prod_equiv_of_closed_compl Submodule.prodEquivOfClosedCompl
 
 /-- Projection to a closed submodule along a closed complement. -/
 def linearProjOfClosedCompl (h : IsCompl p q) (hp : IsClosed (p : Set E))
     (hq : IsClosed (q : Set E)) : E →L[𝕜] p :=
   ContinuousLinearMap.fst 𝕜 p q ∘L ↑(prodEquivOfClosedCompl p q h hp hq).symm
-#align subspace.linear_proj_of_closed_compl Subspace.linearProjOfClosedCompl
+#align subspace.linear_proj_of_closed_compl Submodule.linearProjOfClosedCompl
 
 variable {p q}
 
 @[simp]
 theorem coe_prodEquivOfClosedCompl (h : IsCompl p q) (hp : IsClosed (p : Set E))
     (hq : IsClosed (q : Set E)) :
     ⇑(p.prodEquivOfClosedCompl q h hp hq) = p.prodEquivOfIsCompl q h := rfl
-#align subspace.coe_prod_equiv_of_closed_compl Subspace.coe_prodEquivOfClosedCompl
+#align subspace.coe_prod_equiv_of_closed_compl Submodule.coe_prodEquivOfClosedCompl
 
 @[simp]
 theorem coe_prodEquivOfClosedCompl_symm (h : IsCompl p q) (hp : IsClosed (p : Set E))
     (hq : IsClosed (q : Set E)) :
     ⇑(p.prodEquivOfClosedCompl q h hp hq).symm = (p.prodEquivOfIsCompl q h).symm := rfl
-#align subspace.coe_prod_equiv_of_closed_compl_symm Subspace.coe_prodEquivOfClosedCompl_symm
+#align subspace.coe_prod_equiv_of_closed_compl_symm Submodule.coe_prodEquivOfClosedCompl_symm
 
 @[simp]
 theorem coe_continuous_linearProjOfClosedCompl (h : IsCompl p q) (hp : IsClosed (p : Set E))
     (hq : IsClosed (q : Set E)) :
     (p.linearProjOfClosedCompl q h hp hq : E →ₗ[𝕜] p) = p.linearProjOfIsCompl q h := rfl
-#align subspace.coe_continuous_linear_proj_of_closed_compl Subspace.coe_continuous_linearProjOfClosedCompl
+#align subspace.coe_continuous_linear_proj_of_closed_compl Submodule.coe_continuous_linearProjOfClosedCompl
 
 @[simp]
 theorem coe_continuous_linearProjOfClosedCompl' (h : IsCompl p q) (hp : IsClosed (p : Set E))
     (hq : IsClosed (q : Set E)) :
     ⇑(p.linearProjOfClosedCompl q h hp hq) = p.linearProjOfIsCompl q h := rfl
-#align subspace.coe_continuous_linear_proj_of_closed_compl' Subspace.coe_continuous_linearProjOfClosedCompl'
+#align subspace.coe_continuous_linear_proj_of_closed_compl' Submodule.coe_continuous_linearProjOfClosedCompl'
 
-theorem closedComplemented_of_closed_compl (h : IsCompl p q) (hp : IsClosed (p : Set E))
+theorem ClosedComplemented.of_isClosed_isCompl (h : IsCompl p q) (hp : IsClosed (p : Set E))
```

Comment:
this name is strange, because it can't be used with dot notation. Maybe `IsCompl.closedComplemented_of_isClosed`?

Reply:
It can be used with the new Lean 4 dot notation, see line 141 below. I can add an alias, if you want.